### PR TITLE
John's Pull Request - Settings

### PR DESCRIPTION
**Pull request by jacobburgo**
Settings UI.

### CODE REVIEW

**Review by Jayj2 — APPROVED** [review]

**Review by Jayj2 — APPROVED** [review]
Should all be good to go!

File: app/build.gradle [hunk 1]
```diff
@@ -35,15 +34,21 @@ android {
...
 dependencies {
 
-    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'androidx.core:core-ktx:1.8.0'
     implementation 'androidx.appcompat:appcompat:1.5.0'
     implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-    implementation 'androidx.fragment:fragment-ktx:1.5.2'
+    implementation 'androidx.preference:preference:1.1.+'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.5.2'
```
Comment (jacobburgo) [inline app/build.gradle:51]: Moving forward we need to discuss adding or updating dependencies before doing them. These affect a large amount of files and memory, so we want to all be an agreement about updating/adding them.

File: build.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '7.2.2' apply false
-    id 'com.android.library' version '7.2.2' apply false
+    id 'com.android.application' version '7.3.0' apply false
+    id 'com.android.library' version '7.3.0' apply false
```
Comment (jacobburgo) [inline build.gradle:4]: Same here.

**Review by theRedAces — APPROVED** [review]

File: build.gradle (hunk 2 above)
Comment (Jayj2) [reply build.gradle:4]: Sure thats's not a problem.

**Pull request merged**